blosc/blosc.c

```diff
@@ -1969,14 +1982,9 @@ void blosc_cbuffer_sizes(const void *cbuffer, size_t *nbytes,
                          size_t *cbytes, size_t *blocksize)
 {
   uint8_t *_src = (uint8_t *)(cbuffer);    /* current pos for source buffer */
-  uint8_t version, versionlz;              /* versions for compressed header */
-
-  /* Read the version info (could be useful in the future) */
-  version = _src[0];                       /* blosc format version */
-  versionlz = _src[1];                     /* blosclz format version */
+  uint8_t version = _src[0];               /* version of header */
 
-  version += 0;                            /* shut up compiler warning */
-  versionlz += 0;                          /* shut up compiler warning */
+  assert (version == BLOSC_VERSION_FORMAT); /* Should have been checked earlier */
```

Comment:
Is this comment applicable to a code that is going to be included in the library?  As it is I find it misleading.

Reply:
Yes, I agree the comment is misleading, I will remove the comment. Since there is no way to signal errors I'd like to keep the assert though.